includes/customers/customer-actions.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Customer Actions.
+ *
+ * This file contains all of the first class functions for interacting with a
+ * customer or its related meta data.
+ *
+ * @package     EDD
+ * @subpackage  Actions
+ * @copyright   Copyright (c) 2020, Sandhills Development, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       3.0
+ */
+
+/**
+ * When a new primary address is added to the database, any other primary addresses should be demoted.
+ *
+ * @param string $old_value
+ * @param string $new_value
+ * @param int $item_id
```

Comment:
Can you add some descriptions for what these values are?

`$old_value` should be the previous value of `is_primary` and `$new_value` should be the new value.